SKDREADME.md

```diff
@@ -0,0 +1,22 @@
+# **SIWF SDK**
```

Comment:
I had a thought to create a whole second readme for mobile dev. What do you think?